lisa/sut_orchestrator/azure/tools.py

```diff
@@ -380,3 +380,17 @@ def enable_ssh_on_windows(
             f"-CommandId 'RunPowerShellScript' "
             f"-Parameter @{{'PublicKey'='{public_key_data}'}}"
         )
+
+
+class Azsecd(Tool):
+    @property
+    def command(self) -> str:
+        return "azsecd"
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def install(self) -> bool:
+        self.node.os.install_packages("azure-security")
```

Comment:
self.node.os.install_packages("azure-security")  # type: ignore

Reply:
@rakshaa2000 my bad, forget type check part.

Reply:
Sure @LiliDeng, I'll add that